public/js/lib/devtools-sham/client/shared/key-shortcuts.js

```diff
@@ -141,7 +141,7 @@ KeyShortcuts.parseElectronKey = function (window, str) {
     key = ElectronKeysMapping[key];
     shortcut.keyCode = window.KeyboardEvent[key];
     // Used only to stringify the shortcut
-    shortcut.keyCodeString = key;
+    shortcut.key = key;
```

Comment:
Cool, so it's a bug on m-c too? Strange that we haven't hit it before... I don't see any reason Jaideep can't still contribute to m-c too after his internship :)